pytition/petition/templates/petition/petition_detail.html

```diff
@@ -82,130 +96,130 @@
 </div>
 
 
-<div class="petition-wrapper d-sm-flex align-items-stretch">
-    <div class="content col-sm-6 col-md-8">
+<div class="petition-wrapper">
+    <div class="content">
         <h1>{% autoescape off%}{{ petition.title|striptags }}{% endautoescape %}</h1>
-        <div class="presentation">
-                    {% autoescape off%}{{ petition.text }}{% endautoescape %}
-        </div>
-    </div>
-    <div class="formular col-sm-6 col-md-4" id="petition">
-        <div class="form-wrapper">
-            <p class="intro" id="intro">
-                {% autoescape off%}{{ petition.side_text }}{% endautoescape %}
-                <br/>
-                <span class="sign">Signez la pétition&nbsp;!</span>
-            </p>
-            <div class="counter" id="counter">
-                <p>
-                    Déjà <span id="nb-signatures">{{ petition.signature_number }}</span> signatures.
-                    Objectif : <span class="format-number">{{ petition.target }}</span>
+        <div class="formular" id="petition">
+            <div class="form-wrapper">
+                <p class="intro" id="intro">
+                    {% autoescape off%}{{ petition.side_text }}{% endautoescape %}
+                    <br/>
+                    <span class="sign">Signez la pétition&nbsp;!</span>
                 </p>
-                <div class="progress">
-                    <div class="progress-bar" role="progressbar" id="progress-bar"
-                         data-target="{{ petition.target }}">
+                <div class="counter" id="counter">
+                    <p>
+                        Déjà <span id="nb-signatures">{{ petition.signature_number }}</span> signatures.
```

Comment:
"Déjà" "Objectif" "signatures" : those will need to go through i18n translation.
Maybe in a follow-up PR?

Reply:
Yes, I've noticed many not translatable strings, it should be another PR

Reply:
Ack!